roles/firmware/tasks/install.yml

```
@@ -0,0 +1,23 @@
+# check the timestamps, might want to preserve the old ones
+- name: Backup OS provided Firmware
+  copy:
+    src: "/lib/firmware/brcm/{{ item }}"
+    dest: "/lib/firmware/brcm/{{ item }}.orig"
+  with_items:
+    - brcmfmac43430-sdio.bin
+    - brcmfmac43455-sdio.bin
+
+# grab the old firmware
+- name: Retrieve older firmware
+  get_url:
+    url: "{{ item.url }}"
+    dest: "{{ item.dest }}"
+  with_items:
+    - { url: 'http://download.iiab.io/packages/brcmfmac43430-sdio.bin_2020-02-16_7.45.98.97' dest:/lib/firmware/brcm/brcmfmac43430-sdio.bin.iiab }
```

```suggestion
    - { url: 'http://download.iiab.io/packages/brcmfmac43430-sdio.bin_2018-09-11_7.45.98.65' dest: '/lib/firmware/brcm/brcmfmac43430-sdio.bin.iiab' }
```

...as firmware .97 doesn't help raising the number of clients from 10 back up to 30.  Whereas firmware .65 does, as documented at: https://github.com/iiab/iiab/issues/823#issuecomment-662285202

(Both firmwares require /lib/firmware/brcm/brcmfmac43430-sdio.clm_blob to work.)

In the case of .65, this can be downloaded from: http://d.iiab.io/packages/brcmfmac43430-sdio.clm_blob_2018-09-11_7.45.98.65